docs/troubleshooting/deadline-exceeded-error.mdx

```diff
@@ -40,10 +40,10 @@ If you're using Temporal Cloud, create a [support ticket](/cloud/support#support
 [Frontend Service](/clusters#frontend-service) logs can show which parts of the Cluster aren't working.
 For the error to appear, a service pod or container must be up and running.
 
-OSS users can verify that the Frontend Service is connected and running by using `tctl`.
+OSS users can verify that the Frontend Service is connected and running by using the Temporal CLI.
 
 ```
-tctl --address frontendAddress:frontendPort cluster health
+temporal operator --address frontendAddress:frontendPort cluster health
```

Comment:
```suggestion
temporal operator cluster health --address 127.0.0.1:7233
```

(The `--address` flag no longer needs to be passed to `operator` specifically, and this is a little more readable)